experimental/dds/sequence-deprecated/api-report/sequence-deprecated.alpha.api.md

```diff
@@ -4,12 +4,6 @@
 
 ```ts
 
-export { IJSONRunSegment }
-
-export { SharedSequence }
-
-export { SubSequence }
```

Comment:
`@internal` since we created the tag; why did we have several instances of those showing up here?

Reply:
https://github.com/microsoft/rushstack/pull/5211